test/test.c

```diff
@@ -1144,7 +1145,8 @@ static void ShowHelp()
 "   -v, --verbose <#>       Be verbose (optional argument: logging level)\n"
 "   -Z. --preset <string>   Use a built-in preset. Capitalization matters,\n"
 "                           and if the preset name has spaces, surround it\n"
-"                           with double quotation marks\n"
+"                           with double quotation marks.\n"
+"                           Implies --preset-import-gui.\n"
```

Comment:
I don't understand this addition to help.  -Z currently works without --preset-import-gui.  And it doesn't appear you've modified it to force loading user presets when -Z is specified.